pnpm-lock.yaml

```diff
@@ -13054,6 +13083,26 @@ packages:
   /lines-and-columns/1.2.4:
     resolution: {integrity: sha512-7ylylesZQ/PV29jhEDl3Ufjo6ZX7gCqJr5F7PKrqc93v7fzSymt1BpwEU8nAUXs8qzzvqhbjhK5QZg6Mt/HkBg==}
 
+  /lint-staged/11.2.6:
```

Comment:
also your lock file update is not necessary. please rebase

Reply:
rebased.